src/auto/desktop_app_info.rs

```diff
@@ -91,10 +94,27 @@ pub trait DesktopAppInfoExt: 'static {
         launch_context: Option<&P>,
     );
 
-    //fn launch_uris_as_manager<P: IsA<AppLaunchContext>>(&self, uris: &[&str], launch_context: Option<&P>, spawn_flags: /*Ignored*/glib::SpawnFlags, user_setup: Option<Box<dyn FnOnce() + 'static>>, pid_callback: /*Unimplemented*/FnMut(&DesktopAppInfo, /*Ignored*/glib::Pid), pid_callback_data: /*Unimplemented*/Option<Fundamental: Pointer>) -> Result<(), Error>;
+    fn launch_uris_as_manager<P: IsA<AppLaunchContext>>(
+        &self,
+        uris: &[&str],
+        launch_context: Option<&P>,
+        spawn_flags: glib::SpawnFlags,
+        user_setup: Option<Box<dyn FnOnce() + 'static>>,
+        pid_callback: Option<&mut dyn (FnMut(&DesktopAppInfo, glib::Pid))>,
+    ) -> Result<(), Error>;
 
-    //#[cfg(any(feature = "v2_58", feature = "dox"))]
-    //fn launch_uris_as_manager_with_fds<P: IsA<AppLaunchContext>>(&self, uris: &[&str], launch_context: Option<&P>, spawn_flags: /*Ignored*/glib::SpawnFlags, user_setup: Option<Box<dyn FnOnce() + 'static>>, pid_callback: /*Unimplemented*/FnMut(&DesktopAppInfo, /*Ignored*/glib::Pid), pid_callback_data: /*Unimplemented*/Option<Fundamental: Pointer>, stdin_fd: i32, stdout_fd: i32, stderr_fd: i32) -> Result<(), Error>;
+    #[cfg(any(feature = "v2_58", feature = "dox"))]
+    fn launch_uris_as_manager_with_fds<P: IsA<AppLaunchContext>>(
+        &self,
+        uris: &[&str],
+        launch_context: Option<&P>,
+        spawn_flags: glib::SpawnFlags,
+        user_setup: Option<Box<dyn FnOnce() + 'static>>,
+        pid_callback: Option<&mut dyn (FnMut(&DesktopAppInfo, glib::Pid))>,
+        stdin_fd: i32,
+        stdout_fd: i32,
+        stderr_fd: i32,
```

Comment:
This should be generic over `FromRawFd` for the fds on UNIX and `FromRawHandle` on Windows. Like we do elsewhere.

Passing arbitrary numbers around as fds is unsafe.